gcc/testsuite/rust/compile/macro48.rs

```diff
@@ -0,0 +1,10 @@
+// { dg-excess-errors "allow" }
```

Comment:
hmm we have to be cautious about using dg-excess-errors as it swallows up everything. What are the excess errors?